starky/Cargo.toml

```diff
@@ -34,9 +34,9 @@ plonky = { package="plonky", path = "../plonky", version = "0.0.2" }
 algebraic = { path = "../algebraic" }
 
 #powdr pil compile tool. branch = "main"
-pil_analyzer = {git = "https://github.com/powdr-labs/powdr.git", rev = "eb593dee15e194bd066085022f8a6ea52e2dab9b", package = "pil_analyzer"}
-number = {git = "https://github.com/powdr-labs/powdr.git", rev = "eb593dee15e194bd066085022f8a6ea52e2dab9b", package = "number"}
-ast = {git = "https://github.com/powdr-labs/powdr.git",   rev = "eb593dee15e194bd066085022f8a6ea52e2dab9b",package = "ast"}
+pil_analyzer = {git = "https://github.com/powdr-labs/powdr.git", branch = "main", package = "pil_analyzer"}
```

Comment:
We'd better specify the commit rev, in order to be not influenced by the changes from powdr.